frontend/src/scenes/urls.ts

```diff
@@ -175,19 +167,6 @@ export const urls = {
         `/data-warehouse/settings/${tab ? tab : DataWarehouseTab.ManagedSources}/${id}`,
     annotations: (): string => '/data-management/annotations',
     annotation: (id: AnnotationType['id'] | ':id'): string => `/data-management/annotations/${id}`,
-    projectApps: (tab?: PluginTab): string => `/apps${tab ? `?tab=${tab}` : ''}`,
-    projectApp: (id: string | number): string => `/apps/${id}`,
-    projectAppSearch: (name: string): string => `/apps?name=${name}`,
-    projectAppLogs: (id: string | number): string => `/apps/${id}/logs`,
-    projectAppSource: (id: string | number): string => `/apps/${id}/source`,
-    frontendApp: (id: string | number): string => `/app/${id}`,
-    appMetrics: (pluginConfigId: string | number, params: AppMetricsUrlParams = {}): string =>
-        combineUrl(`/app/${pluginConfigId}/metrics`, params).url,
-    appHistoricalExports: (pluginConfigId: string | number): string => `/app/${pluginConfigId}/historical_exports`,
-    appHistory: (pluginConfigId: string | number, searchParams?: Record<string, any>): string =>
-        combineUrl(`/app/${pluginConfigId}/history`, searchParams).url,
-    appLogs: (pluginConfigId: string | number, searchParams?: Record<string, any>): string =>
-        combineUrl(`/app/${pluginConfigId}/logs`, searchParams).url,
```

Comment:
Is it correct that all these URLs are the ones to go?

Reply:
Yep. Just need to add them as redeircts (doing that now)